viewer/Viewer.cpp

```diff
@@ -140,6 +140,31 @@ namespace Enki
 		}
 	}
 	
+	void ViewerWidget::setCamera (QPointF pos, double altitude, double yaw, double pitch)
+	{
+		this->pos = pos;
```

Comment:
The setCamera method sets attribute camera.

    void ViewerWidget::setCamera(QPointF pos, double altitude, double yaw, double pitch)
    {
        camera.pos = pos;
        camera.altitude = altitude;
        camera.yaw = yaw;
        camera.pitch = pitch;
    }